src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Types/TypeFactory.cs

```diff
@@ -69,16 +66,10 @@ public ArgumentListType CreateArgList()
             return type;
         }
 
-        public ErrorType CreateError(
-            Name name,
-            Name nameText,
-            TypeArray typeArgs)
+        public ErrorType CreateError(Name nameText)
         {
             ErrorType e = new ErrorType();
-            e.SetName(name);
```

Comment:
See previous comment about `GetName()` not being used, and it hence being safe to skip this call to `SetName()`.